docs/content/en/docs/pipeline-stages/init.md

```diff
@@ -64,15 +63,60 @@ deploy:
     - leeroy-web/kubernetes/deployment.yaml
 ```
 
+For projects deploying with `kustomize`, Skaffold will scan your project and look for `kustomization.yaml`s as well as Kubernetes manifests.
+It will attempt to infer the project structure based on the recommended project structure from the Kustomize project: thus, 
+**it is highly recommended to match your project structure to the recommended base/ and overlay/ structure from Kustomize!**
+
+This generally looks like this:
+
+```
+app/      <- application source code, along with build configuration
+  main.go
+  Dockerfile
+...
+base/     <- base deploy configuration
+  kustomization.yaml
+  deployment.yaml
+overlays/ <- one or more nested directories, each with modified environment configuration
+  dev/
+    deployment.yaml
+    kustomization.yaml
+  prod/
+...
+```
+
+When overlay directories are found, these will be listed in the generated Skaffold config as `paths` in the `kustomize` deploy stanza. However, it generally does not make sense to have multiple overlays applied at the same time, so **Skaffold will attempt to choose a default overlay, and put each other overlay into its own profile**. This can be specified by the user through the flag `--default-kustomization`; otherwise, Skaffold will use the following heuristic:
```

Comment:
```suggestion
When overlay directories are found, these will be listed in the generated Skaffold config as `paths` in the `kustomize` deploy stanza. However, it generally does not make sense to have multiple overlays applied at the same time, so **Skaffold will attempt to choose a default overlay, and put each other overlay into its own profile**. The default overlay can be specified by the user through the flag `--default-kustomization`; otherwise, Skaffold will use the following heuristic:
```